README.md

```diff
@@ -271,22 +271,23 @@ Show channel name in hotlist after activity
 ```
 /set weechat.look.hotlist_names_level 14
 ```
-
-Debugging
---------------
+Development
+-----------------
 
 Enable debug mode and change debug level (default 3, decrease to increase logging and vice versa):
+
 ```
-/set plugins.var.python.slack.debug_mode on
-/set plugins.var.python.slack.debug_level 2
+/set plugins.var.python.slack.record_events true
 ```
 
+To run the tests, create a virtualenv and pip install from the `requirements.txt`. Then `pytest` to run them locally.
```

Comment:
This shouldn't be in the middle of the options for debugging. You can move it above, just below the "Development" header.